Resources/Prototypes/Entities/Objects/Fun/toys.yml

```
@@ -599,6 +599,9 @@
       path: /Audio/Effects/bite.ogg
     angle: 0
     animation: WeaponArcBite # Rrrr!
+  - type: Tag
+    tags:
+      - PlushieCarp
```

```suggestion
    tags:
    - Payload
    - ClothMade
    - PlushieCarp
```
you need to add the tags from BasePlushie again since the list is overwritten, not merged